pyproject.toml

```diff
@@ -17,13 +16,14 @@ requires = [
 #
 [project]
 name = "jupyter_server_proxy"
-dynamic = [
-    "authors",
-    "description",
-    "keywords",
-    "urls",
-    "version",
+description = "A JupyterLab extension accompanying the PyPI package jupyter-server-proxy adding launcher items for configured server processes."
```

Comment:
this is now _incorrect_, as this is the thing that shows up on PyPI.

Reply:
This was incorrect before as well I think, but I'll update it by defining a description for the PyPI package.